add(platform): AE updates for networking features (#39)
aiven/aiven-docs: docs/platform/concepts/enhanced-compliance-env.md

Insurance Portability and Accountability Act (HIPAA) and the Payment
Card Industry Data Security Standard (PCI DSS) compliances.

> If you require compliance beyond these, contact the [sales team](mailto:sales@aiven.io)

Reviewer: ```suggestion
If you have other compliance requirements, contact the [sales team](mailto:sales@aiven.io)
```
